include/scrimmage/pubsub/Network.h

```diff
@@ -66,11 +66,19 @@ class Network : public Plugin {
     inline virtual void set_mission_parse(MissionParsePtr mp)
     { mp_ = mp; }
 
+    // Get/set delay values
+    double comm_delay() {return comm_delay_;}
+    void set_comm_delay(double del) {comm_delay_ = del;}
+
  protected:
     RTreePtr rtree_;
     RandomPtr random_;
     MissionParsePtr mp_;
 
+    // Delay handling
+    double comm_delay_ = 0;
```

Comment:
Can you default comm_delay_ to -1. It isn't always safe to check for comm_delay_ == 0 due to the computer precision. Instead, check for comm_delay_ < 0. A negative delay doesn't make sense, so we can disable the comm_delay feature by making the delay negative.